spec/unit/recipes/default_spec.rb

```diff
@@ -2,22 +2,14 @@
 
 describe 'openvpn::default' do
   let(:chef_run) do
-    ChefSpec::SoloRunner.new(step_into: ['openvpn_conf']) do |node|
-      node.set['openvpn']['push_options'] = {
-        'dhcp-options' => ['DOMAIN local',
-                           'DOMAIN-SEARCH local']
-      }
-    end.converge(described_recipe)
+    ChefSpec::SoloRunner.new.converge(described_recipe)
   end
 
   it 'converges' do
     chef_run
   end
+end
 
-  it 'makes a template with dhcp options' do
-    expect(chef_run).to render_file('/etc/openvpn/server.conf')
-      .with_content('push "dhcp-options DOMAIN local"')
-    expect(chef_run).to render_file('/etc/openvpn/server.conf')
-      .with_content('push "dhcp-options DOMAIN-SEARCH local"')
-  end
+describe package('openvpn') do
+  it { should be_installed }
```

Comment:
Cheers! Done in https://github.com/xhost-cookbooks/openvpn/commit/6de23104fcf1e0dd5ab4300d457f75b05f9ed711#diff-598de2b86a1c0b92d822d8633ddfa02cR4.